lib/internal/Magento/Framework/GraphQlSchemaStitching/GraphQlReader.php

```diff
@@ -285,4 +294,58 @@ private function removePlaceholderFromResults(array $partialResults) : array
         }
         return $partialResults;
     }
+
+    /**
+     * Get a module name by file path
+     *
+     * @param string $file
+     * @return string
+     */
+    private static function getModuleNameForRelevantFile($file)
+    {
+        if (!isset(self::$componentRegistrar)) {
+            self::$componentRegistrar = new ComponentRegistrar();
+        }
+        // Validates file when it belongs to default themes
+        foreach (self::$componentRegistrar->getPaths(ComponentRegistrar::THEME) as $themeDir) {
+            if (strpos($file, $themeDir . '/') !== false) {
+                return '';
```

Comment:
it's hard to follow a method with multiple returns, 
try to limit to one return